topics/proteomics/tutorials/metaquantome-function/tutorial.md

```diff
@@ -0,0 +1,424 @@
+
+![MetaQuantome-function-workflow](../../images/mQ-function-workflow.jpg){: width="100%"}
+
+
+> ### Agenda
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+
+# **Pretreatments**
+
+The first step in a tutorial is to get the data from the zenodo link provided and making sure that it is in the correct format.
+
+
+## *Get data*
+
+> ### {% icon hands_on %} Hands-on: Data upload
+>
+> 1. Create a new history for this tutorial and give it a meaningful name
+>
+>    {% include snippets/create_new_history.md %}
+>    {% include snippets/rename_history.md %}
+>
+> 2. Import the files: a Functional File, an Intensity file from [Zenodo]({{ page.zenodo_link }})
+>    or from the shared data library (`GTN - Material` -> `{{ page.topic_name }}` -> `{{ page.title }}`)
+>
+>    ```
+>    https://zenodo.org/record/4110725/files/Function-File.tabular
+>    https://zenodo.org/record/4110725/files/Intensity-File.tabular
+>
+>    ```
+>    {% include snippets/import_via_link.md %}
+>    {% include snippets/import_from_data_library.md %}
+>
+>
+> 3. Rename the datasets (If needed).
+> 4. Check that the datatype ( Make sure they are in the correct formats).
+>  Functional File (format=`tabular`), an Intensity file (format=`tabular`)
+>
+>    {% include snippets/change_datatype.md datatype="datatypes" %}
+>
+> 5. Add to each database a tag corresponding to the name of the input data (optional).
+>
+{: .hands_on}
+
+# **Download metaQuantome Databases**
+
+> ### {% icon hands_on %} Hands-on: Run metaQuantome databases
+>
+> 1. {% tool [metaQuantome: databases](https://toolshed.g2.bx.psu.edu/view/galaxyp/metaquantome_db/7c45aa1c9464) %} with the following parameters:
+>    - {% icon param-select %} *"Databases to Download"*: `Select All` (NCBI Taxonomy Database, Gene Ontology Database and Enzyme Commission (EC) database)
+>
+> 2. Execute.
+>
+>    > ### {% icon comment %} Comment
+>    >
+>    > metaQuantome uses freely available bioinformatic databases to expand your set of direct annotations. 
+>    > 
+>    > 
+>    > The databases are:
+>    > 
+>    > 1. NCBI taxonomy database. consists of a list of all currently identified taxa and the relationships between them.
+>    > 
+>    > 2. Gene Ontology (GO) term database. metaQuantome uses the go-basic.obo file and the metagenomics slim GO (subset of the full GO). More details are 
+>    > available at http://geneontology.org/docs/download-ontology/
+>    > 
+>    > 3. ENZYME database with Enzyme Classification (EC) numbers. This database classifies enzymes and organizes the relationships between them.
+>    > 
+>    {: .comment}
+>
+{: .hands_on}
+
+
+![MetaQuantome:databases](../../images/mq-db.png){: width="100%"}
```

Comment:
please remove all screenshots of tool forms. All information to run the tools should be in the hands-on box